software/chipwhisperer/capture/scopes/_OpenADCInterface.py

```diff
@@ -452,6 +553,9 @@ def lo_gain_errors_disabled(self):
         else:
             return False
 
+    def disable_clip_and_lo_gain_errors(self, disabled):
```

Comment:
This needs to be in `scope.errors` instead of `scope.adc`.

Reply:
Whoops, this is just in the wrong class in this file (needs to be in `TriggerSettings`)

Reply:
Fixed.